satpy/readers/hrit_jma.py

```diff
@@ -188,16 +188,26 @@ def mjd2datetime64(mjd):
 
 
 class HRITJMAFileHandler(HRITFileHandler):
-    """JMA HRIT format reader."""
+    """JMA HRIT format reader.
 
-    def __init__(self, filename, filename_info, filetype_info):
+    By default, the reader computes the exact start time.  As this time is different for every channel,
+    things like angle calculation (SZA correction) can get very slow.  To use approximate times, the user can
```

Comment:
I'd favor "nominal" over "approximate", what do you think?

Reply:
Thanks, I'll use `nominal`. And the double whitespace is a leftover reflex from using Emacs, which thinks wrapping works properly only if there are two whitespaces after a comma...